src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no,initial-scale=1.0, maximum-scale=1.0,minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="product-img"
+        class="card__img"
+      >
+      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__code-item">Product code: 195434</p>
+
+      <div class="card__feedback">
+        <div class="stars stars--1">
+          <div class="stars__box"></div>
+          <div class="stars__box"></div>
+          <div class="stars__box"></div>
+          <div class="stars__box"></div>
+          <div class="stars__box"></div>
+        </div>
+
+        <div class="card__feedback-quantity">
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card__price">
+        <div class="card__price-text">
+          Price:
+        </div>
+        <div class="card__price-amount">
+          $2,199
+        </div>
```

Comment:
the same. check it for whole project

Reply:
thank you for the correction
I will keep it in mind